rcsb/db/wf/RepoLoadWorkflow.py

```diff
@@ -370,20 +377,55 @@ def splitIdList(self, op, **kwargs):
 
         return ok
 
-    def getTimeStampCheck(self, hD, targetFileDir, targetFileSuffix):
+    def getPdbHash(self, pdbid):
+        return pdbid[1:3]
+
+    def getCsmHash(self, pdbid):
+        return os.path.join(pdbid[0:2], pdbid[-6:-4], pdbid[-4:-2])
+
+    def getContentTypePrefix(self, databaseName):
+        if databaseName == "pdbx_core":
+            return "pdb"
+        if databaseName == "pdbx_comp_model_core":
+            return "csm"
+        return ""
+
+    def getTimeStampCheck(self, hD, targetFileDir, targetFileSuffix, databaseName, outputContentType=False, outputHash=False):
         res = hD.copy()
-        for pdbid, value in hD.items():
-            pathToItem = os.path.join(targetFileDir, pdbid + targetFileSuffix)
+        modelPath = None
+        for key, value in hD.items():
             if isinstance(value, dict):
+                # csm
                 timeStamp = value["lastModifiedDate"]
-                value["modelPath"].lower()
+                modelPath = value["modelPath"]
             else:
                 timeStamp = value
+
+            # experimental models are stored with lower case while csms are stored with upper case (except content type)
+            pdbid = key.lower()
+            contentTypePrefix = self.getContentTypePrefix(databaseName)
+            hashPath = self.getPdbHash(pdbid)
```

Comment:
Since this is specific for experimental PDBs, I'd add a conditional above here checking that, e.g. `if databaseName == "pdbx_core"`, just like you do below for CSMs.